### DISCUSSION TITLE
Workflow dispatch api call does not return response data

### DISCUSSION THREAD

**Discussion by vyjadhav1** [community/community#59348] [Apps, API and Webhooks]
### Select Topic Area

Product Feedback

### Body

We have recently migrated our pipeline from Azure devops to Github actions. ADO had api feature where it returns run number after being triggered. However Github action workflow dispatch api call does not return any respose data. Is there any other options or any development being done for this to retuen run number as response data ?

**Comment by ghost** [comment]
We are having the same issue! It would be nice to have a solution here.
